catroid/src/main/AndroidManifest.xml

```diff
@@ -124,34 +126,28 @@
                 <category android:name="android.intent.category.BROWSABLE" />
                 <data
                     android:host="*"
+                    android:mimeType="application/octet-stream"
                     android:pathPattern="@string/catroid_extension_pathPattern"
-                    android:scheme="http" />
+                    android:scheme="file" />
```

Comment:
you can combine both schemes (file and content) in one intent filter this way:

            <intent-filter>
                <action android:name="android.intent.action.VIEW" />
                <category android:name="android.intent.category.CATEGORY_OPENABLE" />
                <category android:name="android.intent.category.DEFAULT" />
                <data android:scheme="content" />
                <data android:scheme="file" />
                <data android:mimeType="application/zip" />
                <data android:host="*" />
                <data android:pathPattern="/.*\\.catrobat" />
            </intent-filter>

Reply:
Thanks will update